libethashseal/Ethash.cpp

```diff
@@ -157,6 +157,8 @@ void Ethash::verifyTransaction(ImportRequirements::value _ir, TransactionBase co
 		}
 		else
 			_t.checkChainId(-4);
+		if (_bi.number() < chainParams().u256Param("metropolisForkBlock") && _t.hasZeroSignature())
```

Comment:
(Make sure then that `_t.checkLowS()` is called only after homestead, because now it's not the case in `SealEngineFace::verifyTransaction`)